fastlane/metadata/android/en-US/title.txt

```diff
@@ -1 +1 @@
-WordPress – Website Builder
+WordPress – Website Builder
```

Comment:
That change correspond to the new copy having a newline at end of file.

This comes from https://github.com/wordpress-mobile/WordPress-Android/pull/17341 and the fact that `android/en-US/*.txt` metadata now come from being copied from `WordPress/metadata/*.txt` files instead of being extracted from the source of `en-GB` back when we relied on a dirty hack for them before (which led to a bug if `en-GB` didn't have a translation for them).